src/commands.ts

```diff
@@ -87,12 +105,20 @@ export const push = async (cmd: { watch: boolean }) => {
           return;
         }
       }
+      if( !cmd.force && await manifestHasChanges() && !await confirmManifestUpdate() ){
```

Comment:
Can you check the formatting? `if (!cmd...`

Reply:
There was no error with lint.
What does the coding rule based on?